### DISCUSSION TITLE
Discord invite in all occurrences (Checked Readme, Docs navbar, Site footer etc.) is invalid

### DISCUSSION THREAD

**Discussion by ummahrican** [open-webui/open-webui#6493] [General]
## Installation Method

N/A was simply checking out this cool project and wanted to join the community! 

## Environment
Probably not too relevant but for posterity, I recreated these issues from Chrome:
Chrome Version 130.0.6723.70 (Official Build) (arm64)

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Interested folks like myself can hop into the Discord community.

## Actual Behavior:

Interested folks like myself can't hop into the Discord community.

## Description

**Bug Summary:**
The invite available on this project is invalid, seems it just needs a permanent invite. Unless this is on purpose; then ignore me! 

## Reproduction Details

**Steps to Reproduce:**
1. Select any of the discord links in the aforementioned places
2. Observe Discord invalid invite error upon redirect.

## Logs and Screenshots

**Browser Console Logs:**
N/A

**Docker Container Logs:**
N/A

**Screenshots/Screen Recordings (if applicable):**
N/A

## Additional Information
N/A

**Comment by silentoplayz** [comment]
Works fine for me.
![image](https://github.com/user-attachments/assets/af070348-e9aa-47d6-ad04-16afca415433)
Here is the invite URL that it opens up to: https://discord.com/invite/5rJgQTnV4s
![image](https://github.com/user-attachments/assets/d0010c85-7254-4457-83aa-18dca9982285)

**Reply by ummahrican** [reply]
<img width="1051" alt="Screenshot 2024-10-27 at 10 34 06 PM" src="https://github.com/user-attachments/assets/d5a1e81a-eade-4d0a-9046-1ee2657fc55f">
This is what I get when clicking but may be pebkac.

**Reply by ummahrican** [reply]
Looks like if I launch it in Safari or incognito it works. May have just been bad caching on my side. Good to close.
